otx/algorithms/segmentation/configs/deeplabv3plus_resnet18/compression_config.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Is this config a copy-paste from other models?

Reply:
Yes, for now it is the same as Lite-HRNet-s and Lite-HRNet-x